docs/docs/getting-started.rst

```diff
@@ -0,0 +1,75 @@
+.. _getting-started:
+
+Getting started
+===============
+
+Where to install Dru?
+---------------------
+
+Dru can be thought as of an engine that provides its clients with answers on blockchain-related queries. As such, it can be run on your local computer, but bear in mind that if the blockchain of a chosen cryptocurrency is large, you need to be equipped with enough hardware (especially RAM). Apart from that, some of the queries can be computationally complex, so this is another factor to consider. This is why it is suggested to have Dru installed on either a server or a workstation that is always on and has enough resources. This will create a typical client-server architecture and many researchers will have the opportunity to use Dru at the same time and gain access to already computed queries' results. Nevertheless, this is not a necessity and you can install Dru on your local computer.
+
+Installing Dru
+--------------
+
+Installing docker
+~~~~~~~~~~~~~~~~~
```

Comment:
Installing docker-compose is also necessary https://docs.docker.com/compose/install/